x-pack/plugins/cases/public/components/cases_context/use_cases_features.tsx

```diff
@@ -9,16 +9,22 @@ import { useMemo } from 'react';
 import { CaseMetricsFeature } from '../../containers/types';
 import { useCasesContext } from './use_cases_context';
 
-interface UseCasesFeatures {
+export interface UseCasesFeatures {
+  isAlertsEnabled: boolean;
   isSyncAlertsEnabled: boolean;
   metricsFeatures: CaseMetricsFeature[];
 }
 
+const isObject = (obj: unknown): obj is object => {
+  return typeof obj === 'object' && obj != null;
+};
+
 export const useCasesFeatures = (): UseCasesFeatures => {
   const { features } = useCasesContext();
   const casesFeatures = useMemo(
     () => ({
-      isSyncAlertsEnabled: features.alerts.sync,
+      isAlertsEnabled: !!features.alerts,
+      isSyncAlertsEnabled: isObject(features.alerts) ? features.alerts.sync : features.alerts,
```

Comment:
I am not entirely convinced of this type change. Can you implement this functionality without adding extra complexity to the `alerts` feature object? keeping the `sync` option or adding a `disabled` key ?

Reply:
I am not sure either. I did not like the `{ sync: true, disabled: true }` signature that is why I made it `{ sync: boolean } | boolean`

Reply:
what about `enabled: true` with true by default? this actually makes more sense than having the type being either an object or boolean.

Reply:
Ok!